goty-server/src/main/kotlin/com/aleinin/goty/submission/SubmissionRepository.kt

```diff
@@ -6,11 +6,14 @@ import java.util.UUID
 
 @Repository
 class SubmissionRepository(private val secretSubmissionRepository: SecretSubmissionRepository) {
-    fun findAllSubmissions() = secretSubmissionRepository.findAll().map { it.toSubmission() }
+    fun findSubmissionYears() = secretSubmissionRepository.findAll().map { it.year }.distinct()
```

Comment:
not really happy with this one. 

This is dead easy to do in Mongo manually

`db.collection.distinct("year")`

But getting that to work in Spring Data was a massive PITA. 

No MongoRepository support. So i tried to make a second MongoTemplate repository to do a manual query but then I ran into issues of converting from BSON to Kotlin `Int` due to missing codecs, and I started to look into codec registering before i hit the eject button